ajax/libs/SoundJS/package.json

```diff
@@ -2,22 +2,31 @@
   "name": "SoundJS",
   "filename": "soundjs.min.js",
   "version": "0.6.0",
-  "description": "A Javascript library that provides a simple API, and powerful features to make working with audio a breeze.",
-  "homepage": "http://www.createjs.com/#!/SoundJS",
+  "description": "A JavaScript library that provides a simple API, and powerful features to make working with audio a breeze. Easily ties in audio file loading to PreloadJS.",
   "keywords": [
-    "sound",
+    "createjs",
+    "soundjs",
+    "gskinner",
+    "javascript",
     "audio",
-    "webAudio"
+    "webaudio",
+    "html5",
+    "flash"
   ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/CreateJS/SoundJS.git"
+    "url": "git+https://github.com/CreateJS/SoundJS.git"
   },
   "license": "MIT",
-  "authors": [
-    "lannymcnie",
-    "OJayRobinson",
-    "gskinner",
-    "wdamien"
+  "author": "gskinner.com, inc.",
+  "homepage": "https://createjs.com/soundjs/",
+  "npmName": "soundjs",
+  "npmFileMap": [
+    {
+      "basePath": "lib",
+      "files": [
+        "**/*"
```

Comment:
Need to modify this to `**/!(README.md)` ( or `**/!(*.md)` if you could put any other markdown file under `dist` in the future) to exclude `README.md`

Reply:
@wdamien Please help modify the pattern to `**/!(*.md)`
Thank you.